x-pack/plugins/fleet/server/plugin.ts

```diff
@@ -430,6 +430,7 @@ export class FleetPlugin
         return new FleetArtifactsClient(core.elasticsearch.client.asInternalUser, packageName);
       },
       fetchFindLatestPackage,
```

Comment:
can't comment higher but should we absorb packageService on line 413 into this new client? `fetchFindLatestPackage` as well.